<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceOptionsPage</class>
 <widget class="QWidget" name="InterfaceOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>543</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="toolbar_show_labels">
        <property name="text">
         <string>Show text labels under icons</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="show_menu_icons">
        <property name="text">
         <string>Show icons in menus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>User interface language:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="ui_language"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_theme">
        <property name="text">
         <string>User interface color theme:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="ui_theme"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="miscellaneous_box">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="toolbar_multiselect">
        <property name="text">
         <string>Allow selection of multiple directories</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="builtin_search">
        <property name="text">
         <string>Use builtin search rather than looking in browser</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="use_adv_search_syntax">
        <property name="text">
         <string>Use advanced query syntax</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="quit_confirmation">
        <property name="text">
         <string>Show a quit confirmation dialog for unsaved changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="filebrowser_horizontal_autoscroll">
        <property name="text">
         <string>Adjust horizontal position in file browser automatically</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="starting_directory">
        <property name="text">
         <string>Begin browsing in the following directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLineEdit" name="starting_directory_path">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="starting_directory_browse">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="ui_theme_container" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toolbar_show_labels</tabstop>
  <tabstop>show_menu_icons</tabstop>
  <tabstop>ui_language</tabstop>
  <tabstop>ui_theme</tabstop>
  <tabstop>toolbar_multiselect</tabstop>
  <tabstop>builtin_search</tabstop>
  <tabstop>use_adv_search_syntax</tabstop>
  <tabstop>quit_confirmation</tabstop>
  <tabstop>filebrowser_horizontal_autoscroll</tabstop>
  <tabstop>starting_directory</tabstop>
  <tabstop>starting_directory_path</tabstop>
  <tabstop>starting_directory_browse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
